eng/templates/Azure.ResourceManager.Template/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0-beta.1 (Unreleased)
 
+### Features Added
+
+This is the first beta release.
+
 ### General New Features
```

Comment:
This can't be here.   The sections need to be restricted to those defined in the [policy](https://azure.github.io/azure-sdk/policies_releasenotes.html)

Reply:
The pipeline reported this error, so add this `Features Added` section for first beta.
![image](https://github.com/Azure/azure-sdk-for-net/assets/20514459/15236b95-71b5-484f-b48b-2428a40c8263)

Reply:
We basically need to redo this changelog and ensure that it follows the policy, using only the approved sections.

Reply:
Fixed